README.md

```diff
@@ -1840,16 +1843,16 @@ The AMT trap does not apply to NSOs.
 
 ### Stock awards vs ISOs vs NSOs
 
-Because the differences are so nuanced, here is a summary of the differences between
-restricted stock awards, ISOs and NSOs from an employee’s point of view.
+Because the differences are so nuanced, what follows is a summary of the differences
+between restricted stock awards, ISOs and NSOs from an employee’s point of view.
 (If you relish tax complexity, you can peruse more
 [here](https://www.irs.gov/taxtopics/tc427.html),
 [here](http://www.startuplawblog.com/2013/05/15/incentive-stock-options-vs-nonqualified-stock-options/),
 [here](http://www.investopedia.com/articles/optioninvestor/07/esoabout.asp), and
-[here](http://www.startuplawblog.com/2010/08/11/top-reasons-nqos-over-isos/)):
+[here](http://www.startuplawblog.com/2010/08/11/top-reasons-nqos-over-isos/).)
 
 - **Restricted stock awards**: Assuming vesting, you pay full taxes early with the 83(b) or at
-  vesting:
+  vesting.
```

Comment:
I'd expect a colon, and thought that was our style. https://github.com/feynmanlabs/editorial-quality#colons

Here and below. I do think the colon adds clarity in a situation like this, as it unambiguously indicates the sub-points elaborate on what was stated.

Reply:
Fixed, but I disagree:

A.) You're right citing the rule, but no style rule is ever hard and fast. That's the first rule of style rules everywhere. In these cases here, a maze of colons is more distracting than helpful (colons after the subjects, colons after a summary of the points that will follow it, colons at each of the three sub-scenarios (X three for each)...). The bullets are clearly elaborating on the summary, as that is what bullets do naturally with or without a colon, and the bullets themselves are making logical use of the colon, which clarifies the structure of the entire section even if the colon is absent initially. Excess of colons = ugly at best, distracting at worst!

B.) These will all be presented in a table at some point, so I'm not that worried either way.